packages/client/hmi-client/src/services/workflow.ts

```diff
@@ -244,7 +244,7 @@ export class WorkflowWrapper {
 				state: {}
 			})),
 
-			status: OperatorStatus.INVALID,
+			status: OperatorStatus.DEFAULT,
 			width: nodeSize.width,
```

Comment:
It might be better to change the banner logic such that
- the operator state is invalid, and
- there is a non-empty output on the operator